modules/command-engine/javadsl/src/main/scala/surge/javadsl/event/SurgeEvent.scala

```diff
@@ -26,17 +27,17 @@ trait SurgeEvent[AggId, Agg, Evt] extends core.SurgeProcessingTrait[Agg, Nothing
 }
 
 object SurgeEvent {
-  def create[AggId, Agg, Evt](businessLogic: SurgeEventBusinessLogic[AggId, Agg, Evt]): SurgeEvent[AggId, Agg, Evt] = {
+  def create[AggId, Agg, Evt](businessLogic: SurgeEventBusinessLogic[AggId, Agg, Evt], ec: ExecutionContext): SurgeEvent[AggId, Agg, Evt] = {
```

Comment:
@jeffboutotte taking it as a regular parameter in the javadsl like here okay right?